library/core/src/ops/control_flow.rs

```diff
@@ -65,3 +65,10 @@ impl<R: Try> ControlFlow<R::Ok, R> {
         }
     }
 }
+
+impl<B> ControlFlow<(), B> {
+    /// It's frequently the case that there's no value needed with `Continue`,
+    /// so this provides a way to avoid typing `(())`, if you prefer it.
+    #[unstable(feature = "control_flow_enum", reason = "new API", issue = "75744")]
+    pub const CONTINUE: Self = ControlFlow::Continue(());
```

Comment:
Comments welcome on whether this is a good idea.

I could also imagine other things like
```
enum ControlFlow<B, C=()> { BreakWith(B), ContinueWith(C) }
impl<B> ControlFlow<B> { pub const Continue: Self = ControlFlow::ContinueWith(()); }
```

But that doesn't seem like how things are done elsewhere...

Reply:
All are intriguing. Also possible would be an associated constant with the exact same name as the variant, since type inference should do a good job of choosing between the enum variant constructor (`fn(()) -> ControlFlow`) and the constant (`ControlFlow`).

I tend to think we should avoid "magic" (a constant that looks like a variant), so I would prefer either the upper-case associated constant or writing out the `(())`.

Reply:
> Also possible would be an associated constant with the exact same name as the variant

I tried this out for fun, and it's weird.  I don't understand why it's allowed (since I thought constants and variants were both in value namespace), but the constant seems to always be hidden: https://github.com/rust-lang/rust/issues/76347

I'll add the `BREAK` equivalent to this PR and ping libs as requested.